bin/e2e-test-integration.js

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env node
```

Comment:
Copied this and `bin/wait-for-build.sh` from the `@woocommerce/e2e-environment` with some small changes. As it didn't allow for making use of puppeteer instead.